cluster-autoscaler/scale_down.go

```diff
@@ -169,25 +174,31 @@ func ScaleDown(
 		glog.V(1).Infof("No node to remove")
 		return ScaleDownNoNodeDeleted, nil
 	}
-	nodeToRemove := nodesToRemove[0]
-	glog.Infof("Removing %s", nodeToRemove.Name)
+	toRemove := nodesToRemove[0]
+	utilization := lastUtilizationMap[toRemove.Node.Name]
+	podNames := make([]string, 0, len(toRemove.PodsToReschedule))
+	for _, pod := range toRemove.PodsToReschedule {
+		podNames = append(podNames, pod.Namespace+"/"+pod.Name)
+	}
+	glog.V(0).Infof("Scale-down: removing %s, utilization: %v, pods: ", toRemove.Node.Name, utilization,
```

Comment:
How about making this more verbose? Like "removing node %s", "pods to be rescheduled"